frontend/src/views/ProjectView.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <ul>
+    <li v-for="[key, file] in files" :key="key">
+      {{ file.filename }}
+    </li>
+  </ul>
+</template>
+<script setup lang="ts">
+import { FileStatus, type File } from '@/constants'
+import { useAppStore, useProjectIdStore } from '@/stores/stores'
+import axios from 'axios'
+import { ref, type Ref } from 'vue'
+
+const storeApp = useAppStore()
+const storeProjectId = useProjectIdStore()
+const props = defineProps<{ initialFiles: FileList | undefined }>()
+const files: Ref<Map<string, File>> = ref(new Map())
+
+if (props.initialFiles == undefined) {
+  const requestedFiles = await getAllFiles(storeProjectId.projectId)
+  requestedFiles.forEach((item) => files.value.set(item.id, item))
+} else {
+  await uploadFiles(props.initialFiles)
+}
+
+async function getAllFiles(projectId: string | null) {
+  var result: File[] = []
+  if (projectId == null) {
+    return result
+  }
+  try {
+    const reponse = await axios.get<File[]>(
+      `${storeApp.constants.env.NAUTILUS_WEB_API}/projects/${projectId}/files`
+    )
+    result = reponse.data
+  } catch (error: any) {
+    console.log(error)
+    storeApp.alertsError(error.message)
+  }
+  return result
+}
+
+async function uploadFiles(uploadFiles: FileList | undefined) {
+  if (uploadFiles == undefined || storeProjectId.projectId == null) {
+    return
+  }
+  const uploadFileRequestsList = []
```

Comment:
Suffixing variable names with the type is not a good practice. And it shouldn't be a list neither

Reply:
Why it shouldn't be a list?  We create promises for each file and call them later, Is it right way?

Reply:
Sorry I changed my mind after looking at the rest in detail. Disregard this